### PR TITLE
[INJIMOB-1709]: Integrate VC verifier aar for Vc verification of Mosip VC

### PR DESCRIPTION
**Pull request by balachandarg-tw**
## Description

> Integrate VC verifier aar for Vc verification of Mosip VC
> Verify RSA Proof VCs using vcverifier library

## Issue ticket number and link

>https://mosip.atlassian.net/jira/software/c/projects/INJIMOB/boards/91?selectedIssue=INJIMOB-1709

### CODE REVIEW

**Review by sree96 — APPROVED** [review]
LGTM

File: shared/Utils.ts [hunk 1]
```diff
@@ -28,7 +28,11 @@ export interface CommunicationDetails {
 }
 
 export const isMosipVC = (issuer: string) => {
-  return issuer === Issuers.Mosip || issuer === Issuers.MosipOtp;
```
Comment (Alka1703) [inline shared/Utils.ts:31]: Please check all other caller of isMosipVC. Adding MosipCertify might start causing issue for other callers.

**Review by Alka1703 — COMMENTED** [review]
I have one more concern. While I was working on this card, I had noticed that the VCMetadat was empty which was causing issue. I do not see any change in that area. Let's connect once to understand this and then merge the PR.

File: shared/Utils.ts [hunk 2]
```diff
@@ -28,7 +28,11 @@ export interface CommunicationDetails {
...
+  return (
+    issuer === Issuers.Mosip ||
+    issuer === Issuers.MosipOtp ||
+    issuer === Issuers.MosipCertify
```
Comment (swatigoel) [inline shared/Utils.ts:34]: We don't need to add this.
We are supported only 4 issuers
1. MosipOtp
2. Mosip
3. StayProtected
4. Mock
We need this hardcoding only for Mosip VC as Mosip VC requires activation which we don't get to know from any property.

File: shared/openId4VCI/Utils.ts [hunk 3]
```diff
@@ -35,6 +35,7 @@ export const Protocols = {
 export const Issuers = {
   MosipOtp: '',
   Mosip: 'Mosip',
+  MosipCertify: 'MosipCertify',
```
Comment (swatigoel) [inline shared/openId4VCI/Utils.ts:38]: don't add MosipCertify

**Review by swatigoel — CHANGES_REQUESTED** [review]

File: shared/openId4VCI/Utils.ts (hunk 3 above)
Comment (balachandarg-tw) [reply shared/openId4VCI/Utils.ts]: Removed.

File: shared/Utils.ts (hunk 2 above)
Comment (balachandarg-tw) [reply shared/Utils.ts]: Removed certify condition

File: shared/Utils.ts (hunk 1 above)
Comment (balachandarg-tw) [reply shared/Utils.ts]: Removed certify condition

**Review by swatigoel — APPROVED** [review]

**Pull request merged**